tes_client/communication/request_sender.py

```diff
@@ -29,7 +29,7 @@
 class RequestSender(Thread):
     """
     Runs as an individual thread to send requests to TesConnection,
-    which then gets routed to TES.  The motivation of the design is different
+    which then gets routed to omega.  The motivation of the design is different
```

Comment:
[Suggestion]
It is better to keep Omega from capital letter.

Reply:
@vitaliybezz Do you mean we should use capital letter? Or we shouldn't use capital letter?

Reply:
I capitalized Omega.